configdata.pm.in

```diff
@@ -131,10 +132,10 @@ use lib '{- $config{builddir} -}';
 use platform;
 _____
 
-        print 'Creating ',$buildfile,"\n";
+        my $tmpl;
```

Comment:
[Nit] Why is this moved up here?

Reply:
This is a stylistic thing, to make it more apparent where this variable is defined, since it's assigned a new value down in (new) line 160.